lib/lita/handlers/totems.rb

```diff
@@ -1,11 +1,15 @@
 require 'lita'
 require 'chronic_duration'
 require 'redis-semaphore'
-
+require "lita/handlers/stats"
 module Lita
   module Handlers
     class Totems < Handler
 
+      
+
+      stats = Stats.new
```

Comment:
We shouldn't need this instance since all the `Stats` methods are class methods.